website_snippet_carousel_product/static/src/scss/s_carousel_product.scss

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+.s_product_carousel {
+    .oe_product {
+        border-width: 0;
+
+        .oe_product_image {
+            position: unset;
+            padding: 15px;
+
+            img {
+                position: unset;
+            }
+        }
+    }
+
+    .carousel-control-prev {
+        left: -15%;
+        .carousel-control-prev-icon {
+            background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='%23000000' viewBox='0 0 8 8'%3E%3Cpath d='M5.25 0l-4 4 4 4 1.5-1.5-2.5-2.5 2.5-2.5-1.5-1.5z'/%3E%3C/svg%3E");
+        }
+    }
+    .carousel-control-next {
+        left: 100%;
+        .carousel-control-next-icon {
+            background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='%23000000' viewBox='0 0 8 8'%3E%3Cpath d='M2.75 0l-1.5 1.5 2.5 2.5-2.5 2.5 1.5 1.5 4-4-4-4z'/%3E%3C/svg%3E");
+        }
+    }
```

Comment:
This is [in standard bootstrap code](https://github.com/odoo/odoo/blob/d4255e01d67154a60cd0c31070ac62e581f50340/addons/web/static/lib/bootstrap/scss/_carousel.scss#L149-L150).

Are you redefining these here for some specific reason?

Reply:
Yes, change the color from white to black.

Reply:
Bootstrap carousel is really used for images... and puts the controls inside the carousel.
The snippet moves these controls to the outside of the carousel (because block mouse interaction with the product tile) and change the colour to black.